### ISSUE TITLE
Email notification configuration is disabled even when ossec_emailnotification is true

### ISSUE THREAD

**Issue by jchenrev** [wazuh/wazuh-puppet#210]
Seems like this line should be `yes` rather than `no` in the template
https://github.com/wazuh/wazuh-puppet/blob/master/templates/wazuh_manager.conf.erb#L7?

**Comment by rshad** [comment]
Hi @jchenrev !

Thank you for reporting this issue, we really appreciate our community contributions. 

My first-look solution is to replace `no` by the value of the variable `ossec_emailnotification ` where this can be `yes` or `no`, and this solution would be as follows:

```ruby
  <%- if @ossec_emailnotification -%>
    <email_notification><%= @ossec_emailnotification %></email_notification>
.
.
```

I'll add this fix as soon as possible.

Kr,

Rshad

**Comment by jchenrev** [comment]
Hi @rshad ,

Thank you for the quick response.

Just want to confirm that whether `$ossec_emailnotification` should be a string or a boolean.

If it's a boolean, then the generated config will be `<email_notification>true</email_notification>`, which seems not to be aligned with the example in the [docs](https://documentation.wazuh.com/3.11/user-manual/manager/manual-email-report/) that uses `<email_notification>yes</email_notification>`.

If it's a string, this [condition](https://github.com/wazuh/wazuh-puppet/blob/master/manifests/manager.pp#L282) (`$ossec_emailnotification == true`) should be changed. (This is another problem, sorry for mentioning it together in this ticket but just a reminder).

**Comment by jchenrev** [comment]
Also if it's a string, the condition in the template should be changed to 
`<%- if @ossec_emailnotification == 'yes' -%>`, right?

**Comment by alanwevans** [comment]
I think @jamebus is correct, I think the real question is:

> whether `$ossec_emailnotification` should be a string or a boolean.

I would submit a PR but I am not sure which way the maintainers would prefer to go.  I think "the puppet" way would be to treat `$ossec_emailnotification` as a `Boolean` and there is precedent in other parts of this module for using `Booleans`.

I've taken a quick stab at both options:
- [As a Boolean](https://github.com/wazuh/wazuh-puppet/compare/master...alanwevans:emailnotification-as-boolean)
- [As a String](https://github.com/wazuh/wazuh-puppet/compare/master...alanwevans:emailnotification-as-string)

Another option would be to refactor to use [Data Types](https://puppet.com/docs/puppet/latest/lang_data_type.html) in the class definitions, which is preferred according to [The Puppet language style guide](https://puppet.com/docs/puppet/latest/style_guide.html#type-signatures), but that is a bigger project than fixing this bug.

**Comment by jm404** [comment]
Fixed in #229
